src/debtManagers/VariableDebtManager.sol

```diff
@@ -0,0 +1,132 @@
+//SPDX-License-Identifier: Unlicensed
+pragma solidity ^0.8.20;
+
+interface IDebtManager {
+    function increaseDebt(address user, uint amount) external;
+    function decreaseDebt(address user, uint amount) external returns(uint);
+    function debt(address market, address user) external view returns (uint);
+    function marketDebt(address market) external view returns(uint);
+}
+
+interface IDbrAMM {
+    function burnDbr(uint exactDolaIn, uint exactDbrBurn) external returns (uint dolaIn);
```

Comment:
Potential case mismatch. It's called `burnDBR` on the HookAMM. Also it doesn't have a return value. Please double check interfaces.